linkerd/admin/src/test/scala/io/buoyant/linkerd/admin/LinkerdAdminTest.scala

```diff
@@ -17,45 +17,57 @@ class BuoyantMainTest extends TwitterServer {
 // TODO: these tests are not getting run
 class LinkerdAdminTest extends FunSuite with Awaits {
 
-  test("serves buoyant admin at /")(new BuoyantMainTest {
-    override def main() {
-      val rsp = await(client(Request("/")))
-      assert(rsp.status == Status.Ok)
+  test("serves buoyant admin at /") {
+    val _ = new BuoyantMainTest {
+      override def main() {
+        val rsp = await(client(Request("/")))
+        assert(rsp.status == Status.Ok)
+      }
```

Comment:
tioli: it would be possible to slightly reduce this boilerplate by creating a mkMain method, but might not be worth the effort:

```scala
def testMain(body: Service[Request, Response] => Unit): Unit = {
  val _ = new BuoyantMainTest {
    override def main(): Unit = {
      body(client)
    }
  }
}
```

and then the tests would look like:

```scala
test("serves buoyant admin at /") {
  testMain { client =>
    val rsp = await(client(Request("/")))
    assert(rsp.status == Status.Ok)
  }
}
```
...not a huge deal either way, though.